### PR TITLE
Modifed typography and button css. Changed font to Source Sans. Updat…

### PR DESCRIPTION
**Pull request by solosapien**
I fixed your stuff.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -63,6 +64,8 @@ header ul li {
     padding: 0 15px;
 }
 header ul li a {
+    letter-spacing: 1px;
+    font-weight: 600;
```
Comment (mattbasta) [inline css/main.css]: Alphabetize

File: css/main.css [hunk 2]
```diff
@@ -130,6 +139,7 @@ header + section p {
 }
 .selling-points div {
     margin: 0 20px;
+    width: 33%;
```
Comment (mattbasta) [inline css/main.css:142]: They're automatically that width thanks to flexbox. Maybe you want to modify the margin?

File: css/main.css [hunk 3]
```diff
@@ -98,12 +105,14 @@ header + section {
     text-align: center;
 }
 header + section h2 {
-    font-size: 28px;
+    font-size: 36px;
+    font-weight:600;
```
Comment (mattbasta) [inline css/main.css]: Space after colon

File: index.html [hunk 4]
```diff
@@ -11,15 +11,18 @@
...
-          <path d="M17.140,12.892 C17.140,13.569 16.603,14.118 15.938,14.118 C15.274,14.118 14.736,13.569 14.736,12.892 C14.736,12.215 15.274,11.666 15.938,11.666 C16.603,11.666 17.140,12.215 17.140,12.892"></path>
-          <path d="M9.881,14.118 C9.221,14.118 8.685,13.576 8.679,12.902 C8.668,11.601 8.996,10.311 9.628,9.174 C10.590,7.441 12.155,6.196 14.036,5.665 C15.917,5.135 17.888,5.382 19.586,6.363 C20.410,6.838 21.124,7.460 21.709,8.212 C22.121,8.742 22.035,9.513 21.514,9.934 C20.996,10.355 20.238,10.266 19.826,9.736 C19.434,9.232 18.955,8.815 18.402,8.496 C17.262,7.838 15.941,7.671 14.677,8.027 C13.416,8.384 12.365,9.220 11.721,10.382 C11.296,11.145 11.076,12.010 11.083,12.882 C11.089,13.558 10.554,14.112 9.891,14.118 L9.881,14.118 Z"></path>
-          <path d="M1.202,18.601 C0.538,18.601 0,18.052 0,17.375 C0,16.699 0.538,16.150 1.202,16.150 L2.704,16.150 C3.368,16.150 3.906,16.699 3.906,17.375 C3.906,18.052 3.368,18.601 2.704,18.601 L1.202,18.601 Z"></path>
+      <svg width="118" height="19" viewBox="0 0 118 19" xmlns="http://www.w3.org/2000/svg">
+        <title>Pinecast</title>
```
Comment (mattbasta) [inline index.html:15]: Ugh

File: index.html [hunk 5]
```diff
@@ -130,6 +136,7 @@
 (function() {
   var atTop = true;
   var header = document.querySelector('header');
+  var logotype = document.querySelector('g.logotype');
```
Comment (mattbasta) [inline index.html]: Unused?

File: css/main.css (hunk 2 above)
Comment (solosapien) [inline css/main.css:142]: Without a width specified flexbox will only make sure that the objects all fit within one row. The width call forces them to display as three equal items. Attached are screenshots of how it displays without that call.
screen shot 2016-01-20 at 9 44 59 pm
screen shot 2016-01-20 at 9 44 50 pm

File: index.html (hunk 5 above)
Comment (solosapien) [inline index.html]: Yup. Good catch!

**Comment by mattbasta** [comment]
R+

**Pull request merged**